config/index.js

```diff
@@ -379,7 +379,7 @@ var conf = convict({
       doc: 'Key prefix for session tokens in Redis'
     },
     maxConnections: {
-      default: 10,
+      default: 100,
```

Comment:
I upped this because we're obviously going to use many more connections now that Redis may be updated from any endpoint that is authenticated by session token.

Is 100 a sensible value in our prod environment? No idea.

/cc @jrgm, @jbuck